Formula/moarvm.rb

```diff
@@ -0,0 +1,41 @@
+class Moarvm < Formula
+  desc "VM for NQP And Rakudo Perl 6"
+  homepage "https://moarvm.org"
+  url "https://moarvm.org/releases/MoarVM-2018.11.tar.gz"
+  sha256 "b0f378fb72dac6fa2af2ab9c0dbd8012fb9bc43d223a7583c4ecb68b17ffa7ea"
+  head "https://github.com/MoarVM/MoarVM.git"
+
+  depends_on "make" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libatomic_ops"
+  depends_on "libffi"
+  depends_on "libtommath"
+  depends_on "libuv"
+
+  def install
+    libffi = Formula["libffi"]
```

Comment:
You don't need to do this, you can place `Formula...` directly in the code.

Reply:
i left `libffi` in because it's being called so much